src/docs/product/sentry-basics/enrich-data/index.mdx

```diff
@@ -26,11 +26,11 @@ The SDK allows you to set various types of data that are then attached to every
 
 ## Using Your Data {#utilizing-your-data}
 
-Adding custom **structured data** that is unique to your applications, users, and business logic enriches your error data and provides valuable context to every error. You can also use that same data to customize Sentry workflows and adapt them to your precise needs. Let's look at those major workflows:
+Adding custom **structured data** that is unique to your applications, users, and business logic enriches your data and provides valuable context to every event. You can also use that same data to customize Sentry workflows and adapt them to your precise needs. Let's look at those major workflows:
```

Comment:
This isn't an edit for this line but for line 21 where it talks about structured data making it so you can query across all errors. Should that be changed to all events?

Reply:
Yep, done!